sormas-api/src/main/resources/captions.properties

```
@@ -141,6 +141,8 @@ actionSettings=User Settings
 actionNewForm=New Form
 actionOverwrite=Overwrite
 actionRemindMeLater=Remind me later
+actionUnclearLabMessage=Mark as unclear
+actionManualForwardLabMessage=Mark as manually forwarded
```

I find this slightly confusing. Actually, the lab message is just marked as forwarded, not as MANUALLY forwarded. The status makes no distinction between forwarded via S2S and manually forwarded. I would therefore suggest to change this to 
```suggestion
actionManualForwardLabMessage=Mark as forwarded
```